src/wasm-binary.h

```diff
@@ -116,6 +116,7 @@ struct LEB {
   }
 };
 
+#define SIZE32LEB(v) U32LEB((uint32_t)(v))
```

Comment:
That is useful, but I think the real question is where a non-32-bit value comes from. It's an array or map, perhaps those types should return 32-bit values anyhow, as mentioned before.